src/components/App.js

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
+import styled from 'styled-components';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
 
-const App = () => (
-  <div>This is my calculator</div>
-);
+
+const AppContainer = styled.div`
+display: flex;
+flex-direction: column;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 700px;
+
+  @media (max-width: 768px) {
+    width: 500px;
+  }
+`;
+
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component {
+  render() {
+    return (
+      <AppContainer id="calculator">
+        <Display />
+        <ButtonPanel />
+      </AppContainer>
+    );
+  }
+}
```

Comment:
I think you can use a function component for this milestone until the component becomes stateful. Also, don't forget to remove this disabled rule when your component becomes stateful in future milestones.